bin/techsupport

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -e
+
+TMPDIR=./techsupport
+mkdir -p $TMPDIR
+
+echo Collecting system information
+sudo ovs-vsctl show > $TMPDIR/ovs_vsctl_show.txt
+lsb_release -a > $TMPDIR/lsb_release.txt 2>/dev/null
+ip route > $TMPDIR/ip_route.txt
+ip link show > $TMPDIR/ip_link_show.txt
+arp -n > $TMPDIR/arp.txt
+
+if [ -d inst ]; then
+    echo Collecting run logs and config from 'inst' and 'local'
+    cp -r inst $TMPDIR
+    cp -r local $TMPDIR
```

Comment:
There may be gcp credentials in local which should be omitted?

Reply:
good point, omitting them now

Reply:
I remembered that there's also cretendials places in inst/